.eslintrc.js

```diff
@@ -1,26 +1,19 @@
 module.exports = {
-    parser: "@typescript-eslint/parser", // Specifies the ESLint parser
-    extends: [
-        "plugin:@typescript-eslint/recommended" // Uses the recommended rules from the @typescript-eslint/eslint-plugin
-    ],
-    parserOptions: {
-        ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
-        sourceType: "module" // Allows for the use of imports
-    },
-    plugins: ["prettier"],
-    rules: {
-        // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
-        // e.g. "@typescript-eslint/explicit-function-return-type": "off",
-        "@typescript-eslint/no-explicit-any": "error",
-        "@typescript-eslint/func-call-spacing": "error",
-        "@typescript-eslint/indent": ["error", 2],
-        "@typescript-eslint/member-ordering": "error",
-        "@typescript-eslint/semi": "error",
-        "@typescript-eslint/ban-ts-ignore": "off",
-        "max-len": ["error", {
-            "code": 120
-        }],
-        "prefer-const": "error",
-        "sort-keys": "error"
-    }
-};
\ No newline at end of file
+  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'prettier'],
+  parserOptions: {
+    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
+    sourceType: 'module' // Allows for the use of imports
+  },
+  plugins: ['@typescript-eslint', 'prettier'],
+  rules: {
+    // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
+    // e.g. "@typescript-eslint/explicit-function-return-type": "off",
+    '@typescript-eslint/ban-ts-ignore': 'off',
+    '@typescript-eslint/func-call-spacing': 'error',
+    '@typescript-eslint/member-ordering': 'error',
+    '@typescript-eslint/no-explicit-any': 'error',
+    'prefer-const': 'error',
+    'sort-keys': 'warn'
+  }
```

Comment:
I think it's already hard to read the amount of changes including the formatting. If I add the new eslint config it will create even more changes. So it's better to tackle it in the separate PR

Reply:
It will still support existing prettier, so it wont change formatting

Reply:
It's not working out of the box, I created the PR to update the libraries first https://github.com/ChainSafe/eslint-config/pull/17